.github/workflows/github-ci.yml

```
@@ -55,6 +56,8 @@ jobs:
   nix-build:
     name: '${{ matrix.os }} ${{ matrix.build_type }} shared=${{ matrix.shared_lib }}'
     runs-on: '${{ matrix.os }}'
+    permissions: 
```

```suggestion
    permissions:
```
removed trailing space after colon.